src/styles.scss

```diff
@@ -499,6 +499,15 @@ textarea:focus {
   right: 20px;
 }
 
+.inline-flex-form {
+  display: flex !important;
```

Comment:
I always shudder when I see `!important`, especially in a global stylesheet.

But my CSS is pretty rusty, and I'm not as up-to-date with the new way of doing things. Will defer to Gary's judgement.

Reply:
Yep I'm usually iffy about `!important` too, but my excuse is it's exactly how Bootstrap 4.1 does it: https://getbootstrap.com/docs/4.1/utilities/flex/#enable-flex-behaviors